src/mono/CMakeLists.txt

```diff
@@ -283,6 +283,7 @@ elseif(CLR_CMAKE_HOST_OS STREQUAL "windows")
     add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/GF>) # enable string pooling
```

Comment:
Nit: Control flow guard is not an optimization. I would enable it for all build flavors, not just for Release.